tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -60,6 +60,10 @@ def _run(self):
                     SyncObject(data=json.dumps(raw_data))
                 sync_object.save()
 
+                gluster_brick_dir = NS.gluster.objects.\
+                                     GlusterBrickDir()
+                gluster_brick_dir.save()
```

Comment:
That is true. But I was thinking of having a clear separation between tendrl defaults and user configuration. Hence the defaults in a read-only object with the user/system configuration in a separate object that's subjected to sync. @r0h4n what do you think?